components/MenuEditorials/MenuItem.js

```diff
@@ -11,13 +11,20 @@ import {GroupSection, GroupTitleContainer, GroupContent, Hyperlink} from './styl
 
 const groupTitleDefaultProps = {
   color: 'neutral2',
-  element: 'h4',
+  element: 'a',
```

Comment:
Acho que aqui não é a. Pode ser um span

```suggestion
  element: 'span',
```